### PR TITLE
[hail.utils]Added examples for Struct.drop

### PR DESCRIPTION
**Pull request by CDiaz96**

### CODE REVIEW

File: hail/python/hail/utils/struct.py [hunk 1]
```diff
@@ -166,6 +166,23 @@ def drop(self, *args):
         -------
         :class:`.Struct`
             Struct without certain fields.
+
+        Examples
+        --------
+
+        Define a Struct `s`
+
+         >>> s = hl.Struct(food=8, fruit=5, bar=2, apple=10)
```
Comment (jigold) [inline hail/python/hail/utils/struct.py:175]: I think the alignment of the code examples with lines starting with `>>>` and the result shown below need to be aligned with the text above.

Comment (danking) [inline hail/python/hail/utils/struct.py:175]: I think you have one extra space before all your code blocks.

File: hail/python/hail/utils/struct.py [hunk 2]
```diff
@@ -166,6 +166,23 @@ def drop(self, *args):
...
+
+        Drop one field from `s`
+
+         >>> s.drop(bar=2)
```
Comment (danking) [inline hail/python/hail/utils/struct.py:179]: When we drop, we just drop the name of the field: `s.drop('bar')`.

File: hail/python/hail/utils/struct.py [hunk 3]
```diff
@@ -166,6 +166,23 @@ def drop(self, *args):
...
+         Struct(food=8, fruit=5, apple=10)
+
+        Drop two fields from `s`
+
+         >>> s.annotate(food=8, fruit=5)
```
Comment (danking) [inline hail/python/hail/utils/struct.py:184]: looks like this is an annotation instead of a drop!

**Review by danking — DISMISSED** [review]

File: hail/python/hail/utils/struct.py (hunk 2 above)
Comment (jigold) [inline hail/python/hail/utils/struct.py:179]: I think this is not correct. What is the argument type `drop` requires? Remember drop is deleting a field. We don't need to assign a value at the same time.

File: hail/python/hail/utils/struct.py (hunk 3 above)
Comment (jigold) [inline hail/python/hail/utils/struct.py:184]: See comment above and this should be `delete` and not `annotate`.

**Review by jigold — DISMISSED** [review]

File: hail/python/hail/utils/struct.py (hunk 3 above)
Comment (CDiaz96) [reply hail/python/hail/utils/struct.py]: Fixed

Comment (CDiaz96) [reply hail/python/hail/utils/struct.py]: Switched to `drop`

**Review by danking — APPROVED** [review]

**Pull request merged**